swagger.yaml

```diff
@@ -270,9 +270,64 @@ paths:
         - pass_auth:
             - write:token
             - read:token
+
+  /account/gatekeeperNetwork:
+    get:
+      tags:
+        - gatekeeperNetwork
+      summary: Retrieve a list of all your organization's custom pass types ( gatekeeper networks ).
+      description: Retrieve the full list of all custom pass types ( also called gatekeeper networks ) managed by your organization
```

Comment:
don't think we need 'all' if we already say 'full list'

Reply:
fixed